.gitignore

```diff
@@ -12,4 +12,5 @@
 *.out
 .idea/
 .vscode/*
-dist/
\ No newline at end of file
+dist/
+vendor/
```

Comment:
But if you use go mod now (GO111MODULE=on), then `vendor/` should not be used?

Reply:
removed from the makefile